packages/cloudwatch-metrics/__tests__/index.js

```diff
@@ -93,14 +93,12 @@ test.serial(
     handler
       .use(
         metrics({
-          dimensions: [
-            {
-              Runtime: 'NodeJS',
-              Platform: 'ECS',
-              Agent: 'CloudWatchAgent',
-              Version: 2
-            }
-          ]
+          dimensions: {
+            Runtime: 'NodeJS',
+            Platform: 'ECS',
+            Agent: 'CloudWatchAgent',
+            Version: 2
+          }
```

Comment:
don't we need a test for an array? If an array fails this would be considered a breaking change and would have to wait for v5.

Reply:
Just added a second test there for an array of objects